Duplicati/Library/Backend/Mega/Strings.cs

```diff
@@ -6,8 +6,10 @@ internal static class MegaBackend {
         public static string AuthPasswordDescriptionShort { get { return LC.L(@"Supplies the password used to connect to the server"); } }
         public static string AuthUsernameDescriptionLong { get { return LC.L(@"The username used to connect to the server. This may also be supplied as the environment variable ""AUTH_USERNAME""."); } }
         public static string AuthUsernameDescriptionShort { get { return LC.L(@"Supplies the username used to connect to the server"); } }
+        public static string AuthTwoFactorKeyDescriptionShort { get { return LC.L(@"Supplies the two-factor key used to connect to the server"); } }
+        public static string AuthTwoFactorKeyDescriptionLong { get { return LC.L(@"The two-factor authentication key used to connect to the server."); } }

```

Comment:
I think "used to connect to the server" could be a little confusing to users that are new to 2FA and TOTP codes.  Perhaps we can improve this a little by saying something like

Short:
```
The shared secret used to generate two-factor TOTP codes.
```

Long:
```
For accounts with two-factor authentication enabled, this is the shared secret used to generate the two-factor TOTP codes.
```

Reply:
I agree with you and I made the changes.